getting-started/installation-linux.rst

```diff
@@ -252,6 +252,16 @@ Now, run
 
     make
 
+If the build fails due to implicit delcaration of ``xdr_*`` functions it is likely async that should build against libtirpc. To do so you can uncomment ``# TIRCP=YES`` in ``configure/CONFIG_SITE`` of asyn, such that it states:
```

Comment:
typos: delcaration -> declaration, async -> asyn, TIRCP -> TIRPC
missing comma after "To do so"

Reply:
oops :-) Thanks

Reply:
I've fixed it.